contrib/translations/DE.json

```diff
@@ -0,0 +1,28 @@
+{
+	"language": "DE",
+	"list": [
+	{
+		"1": "Keine Benutzer- oder Systemkonfigurationsdatei gefunden, die Anwendungsstandardwerte verwendet",
```

Comment:
Hi, 
I just saw this PR, did you by chance create the german translation automatically?
I think at least the first log message sounds a bit off. 
I think this sounds better:
"Keine Benutzer- oder Systemkonfigurationsdatei gefunden, es werden die Standardwerte verwendet"

Reply:
@oohjio
Many thanks for looking at this PR in its very early stage of development.

This is a WIP at the moment. The DE.json is just a 'google' translate of the first few messages, so that I can ensure, when building the PR (as it is not complete), is able to provide the right 'translations' for the language being selected. I could have also used dummy text (eg 'asdfasdfasfdasfdasdf') .. but I didnt.

When this PR is ready, there will only be the EN-US.json file, with the community able to provide a language.json file, which then can be used / loaded into the application for folks to use.

Please keep an eye on this, and if you are interested in providing the DE.json when the main code changes are done, please create / update the file so that this feature can launch with accurate & correct language translations for Deutsche.

Reply:
@abraunegg 
Thanks for the reply. 
I see, that makes sense, 
Will keep an eye on this.